src/assets/icons/BackIcon.jsx

```diff
@@ -0,0 +1,10 @@
+export function BackIcon(props) {
```

Comment:
Where do you get these svgs from?
I use react-icons, do you just copy the svg by inspecting them or what?

Reply:
will ping you the link